src/app/eth-memento/module/component/txsGrid/TxTypeExtraRenderer.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from "react";
+import { IGridFieldRenderer } from "@alethio/ui/lib/control/grid/state/IGridFieldRenderer";
+import { ITranslation } from "plugin-api/ITranslation";
+import { ITxLiteByAccount } from "app/eth-memento/data/tx/byAccount/ITxLiteByAccount";
+import styled from "@alethio/explorer-ui/lib/styled-components";
+import { TooltipRegular } from "@alethio/ui/lib/overlay/tooltip/TooltipRegular";
+import { ErrorSmallIcon } from "@alethio/ui/lib/icon/ErrorSmallIcon";
+
+const IconWrapper = styled.div`
+    position: absolute;
+    left: -8px;
+    top: -4px;
+    transform: translateX(-100%);
+`;
+
+export class TxTypeExtraRenderer implements IGridFieldRenderer<ITxLiteByAccount> {
+
+    constructor(private translation: ITranslation) {}
+
+    render(f: ITxLiteByAccount) {
+        return <div style={{ position: "relative" }}>
+            { f.error === false ?
+            <IconWrapper>
```

Comment:
Would be better to have a status renderer and a status column instead of the type column. Another posibility would be to piggyback on the hash column

Reply:
Resolved by creating a renderer that wraps the TxLinkRenderer and adds a status absolute positioned. Like you said "piggyback on the hash column"